packages/cosmic-proto/package.json

```diff
@@ -148,9 +148,9 @@
     "!CHANGELOG.md"
   ],
   "scripts": {
-    "build": "tsc --project tsconfig.build.json",
+    "build": "tsc --project tsconfig.build.json && node scripts/minify.cjs",
```

Comment:
do we still need this minifying? 
I think we should keep that change separate.

Reply:
I removed commits for cosmic-proto minification since it is no longer necessary in aggregate with esbuild.